mayastor/tests/add_child.rs

```diff
@@ -0,0 +1,129 @@
+use mayastor::{
+    bdev::{nexus_create, nexus_lookup, ChildStatus},
+    core::{mayastor_env_stop, MayastorCliArgs, MayastorEnvironment, Reactor},
+};
+use std::{ops::Deref, process::Command};
+
+static NEXUS_NAME: &str = "nexus";
+
+static DISKNAME1: &str = "/tmp/disk1.img";
+static BDEVNAME1: &str = "aio:///tmp/disk1.img?blk_size=512";
+
+static DISKNAME2: &str = "/tmp/disk2.img";
+static BDEVNAME2: &str = "aio:///tmp/disk2.img?blk_size=512";
+
+pub mod common;
+
+fn test_start() {
+    common::mayastor_test_init();
+    // Create disk images
+    let disks = [DISKNAME1, DISKNAME2];
+    disks
+        .iter()
+        .map(|disk| {
+            let output = Command::new("truncate")
+                .args(&["-s", "64m", disk])
+                .output()
+                .expect("failed to create disk image");
+
+            assert_eq!(output.status.success(), true);
+        })
+        .for_each(drop);
```

Comment:
you can swap out the `.map()` for the `.for_each()` directly IOW:
```rust
disk.iter().for_each(|disk| { ... })
```

Reply:
:+1: